packages/cli/src/commands/web/route/create.ts

```diff
@@ -0,0 +1,63 @@
+import { GluegunCommand } from "@webstone/gluegun";
+
+const command: GluegunCommand = {
+  alias: ["c"],
+  description: "Create a new web page",
+  run: async (toolbox) => {
+    const { parameters, print, prompt, strings, template } = toolbox;
+
+    let name = parameters.first;
+    if (!name) {
+      const result = await prompt.ask({
+        type: "input",
+        name: "name",
+        message: `What's the page name?`,
+      });
+      if (result && result.name) name = result.name;
+    }
+
+    if (!name) {
+      print.error("Please provide a page name.");
+      return;
+    }
+
+    const typesPrompt = (await prompt.ask({
+      type: "multiselect",
+      name: "types",
+      message: "What types of page do you want to create?",
+      choices: [
+        "+page.svelte",
```

Comment:
Do you think we should read the directory first and add an annotation to files in case they already exist?

Reply:
Yes let's do that. Even better, if we can disable the choices for files that already exist, let's do that too. Lastly, what happens if we can't disable choices and someone selects a file that already exists?

I'd argue we log a warning and ignore that file, keeping the existing one. What do you think?